documentation/src/main/asciidoc/entity-view/manual/en_US/01_general.adoc

```diff
@@ -0,0 +1,21 @@
+:numbered!:
+
+[preface]
+== Preface
+
+JPA applications that use the entity model in every layer often suffer from the infamous `LazyInitializationException` or _N + 1_ queries issues.
+This is mainly due to the use of a too general model for a use case and is often solved by making use of a specialized DTO and adapting queries to that structure.
+The use of DTOs normally requires adapting many parts of an application and a lot of boilerplate code which is why people tend to do _the wrong thing_ like making use of
+the _open session in view_ anti-pattern. Apart from lazy loading issues, also the performance suffers due to selecting _unnecessary data_ that a UI is never displaying.
+
+{projectname} entity views tries to solve these and many more problems a developer faces when having to implement efficient model mapping in a JPA application.
+It allows to define DTOs as interfaces and provide the mappings to the JPA model via annotations. It has smart defaults(convention-over-configuration) allowing to omit most mappings.
+The DTOs are applied to a query builder through the `ObjectBuilder` extension point making it possible to separate querying logic from the projections but still have high performance queries.
```

Comment:
Better:
`By applying DTOs to a query builder through the `ObjectBuilder` extension point it is possible to separate query logic from the projections while still enjoying high performance queries.`